bin/bridge-client/src/modes/operator/task_manager.rs

```diff
@@ -278,6 +284,58 @@ where
     Ok(())
 }
 
+async fn update_failed_status_in_db(
+    duty_status_ops: &Arc<BridgeDutyOps>,
+    tracker_txid: Txid,
+    exec_err: &ExecError,
+    max_retry_count: u32,
+) {
+    let status = match duty_status_ops.get_status_async(tracker_txid).await {
+        Ok(status) => status,
+        Err(db_err) => {
+            error!(%db_err, %tracker_txid, status="failed", "and could not update status in db either");
```

Comment:
This log line is really weird.  Was there some revisions that were happening that got munged?

Reply:
This was refactored out from another function where we first notice the error and log the error in the database. So, the log message on the console would read as follows:

```
--- could not do some action
--- and could not update the status in db either
```

Even though it has been refactored out, I still think it makes sense for the logs to tell a story.